pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -958,7 +969,8 @@ func (m *ManagerImpl) callPreStartContainerIfNeeded(podUID, contName, resource s
 
 	m.mutex.Unlock()
 	devs := devices.UnsortedList()
-	klog.V(4).InfoS("Issuing a PreStartContainer call for container", "containerName", contName, "podUID", podUID)
+
+	klog.V(4).InfoS("Issuing a PreStartContainer call for container", "containerName", contName, "podUID", string(podUID))
```

Comment:
@tliu2021 Would it make sense to use the same logging approach here and in a lot of other places in the device manager code? Otherwise the PR looks half-backed.

Reply:
My (unsolicited) take is that we should get a general consensus / ack (or at least no-objections) about the approach before to fix all the managers. I'm proposing https://github.com/kubernetes/kubernetes/pull/120569 as working example. I'd love to start a conversation but I'm struggling to find the best forum. E.g. should sig-instrumentation be involved? is a slack thread sufficient?
Perhaps the best way forward is to gather examples of the new logging and use these examples as conversation starter